docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -121,6 +121,21 @@ $ tctl create github.yaml
   able to determine team memberships for these organizations.
 </Admonition>
 
+After authentication Teleport will create a temporary GitHub type user within Teleport for the duration of the 
+session. The GitHub username is automatically added to that user's `internal.logins` variable
```

Comment:
Technically that's a _trait_, not a variable, right?

Reply:
The `internal.logins` is the trait to the user.  Within here we refer to `internal.logins` as a variable for its usage in roles.  I've updated to describe as a trait variable that I believe is more clear.